docs/development/build-and-run-on-mac.md

```diff
@@ -0,0 +1,292 @@
+# Setting up Waltz development environment on MacBook with PostgreSQL
+Welcome! This guide is to walk you through setting up your development environment on MacOS.
+
+This document describes how to:
+1. install Homebrew package manager
+1. install git
+1. install Java Development Kit 8 (OpenJDK)
+1. install maven
+1. install Node.js
+1. install PostgreSQL and create the Waltz database
+1. configure maven profile for PostgreSQL database connection
+1. build Waltz
+1. run Waltz
+
+This guide is made on 3rd February, 2023 with all versions of the components being latest at this point in time. As you may use this guide later in time, the versions can be different, but the following order and logic will be still applicable.
+
+# Step 1. Install Homebrew
+To install required dependencies on MacOS, use the Homebrew package manager. To check wether Homebrew is installed on your local machine, run the `brew --version` command in your Terminal
+msagi@miklos-mbp ~ % brew --version
```

Comment:
Thank you for all the detail provided, I'm sure this is going to be incredibly useful!

 One small change, could the reference to your cmd prompt "msagi@miklas-mbp" be renamed to "user@macbook" or similar?

Reply:
Done!